pom.xml

```diff
@@ -60,14 +60,21 @@ THE SOFTWARE.
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <java.level>8</java.level>
-    <jenkins.version>2.164.3</jenkins.version>
+    <jenkins.version>2.176.2</jenkins.version>
     <jetty.version>9.4.5.v20170502</jetty.version>
     <!--TODO: Reenable once all the issues are fixed (JENKINS-57353)-->
     <findbugs.failOnError>false</findbugs.failOnError>
   </properties>
 
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <groupId>io.jenkins.tools.bom</groupId>
+        <artifactId>bom</artifactId>
+        <version>2.176.2</version>
```

Comment:
So this takes the version in the specified BOM instead of the ones in this project POM right? Which version are included in such BOM, the ones necessary for Jenkins to run properly?

What about the dependencies that are not needed for Jenkins but for JFR itself?

Reply:
Basically the BOM provides a list of Pipeline plugins which are known to be compatible with each other. This is the main benefit, because maintaining Pipeline plugin dependencies is a nightmare now

Jenkins Core BOM is a separate story ATM, can link some internal chats in PM

Reply:
```suggestion
        <!--WARNING: it is not a Jenkins core version. https://issues.jenkins-ci.org/browse/JENKINS-58770 -->
        <version>2.176.2</version>
```